config/state_machine/configs/00_moj/20_sar/20_sar_standard_workflow.yml

```diff
@@ -15,6 +15,9 @@
                   switch_workflow: trigger
                 link_a_case:
                 remove_linked_case:
+                request_further_clearance:
+                  if: Case::SARPolicy#can_request_further_clearance?
+                  switch_workflow: trigger
```

Comment:
I just realised that these changes are being made to SAR, but of course won't affect Overturned SAR cases. @stephenrichards and I just discussed how we might address this, so I think we can leave it for now. We'll make a new ticket to make it so that Overturned SAR uses the SAR state machine.